plugins/kubernetes/app/assets/javascripts/kubernetes/controllers/kubernetes_roles_ctrl.js

```diff
@@ -1,5 +1,16 @@
-samson.controller('KubernetesRolesCtrl', function($state, $stateParams) {
+samson.controller('KubernetesRolesCtrl', function($scope, $stateParams, kubernetesService, kubernetesRoleFactory) {
+  $scope.project_id = $stateParams.project_id;
 
-//TODO: Load roles asynchronously
+  $scope.roles = [];
 
+  (function loadKubernetesRoles() {
```

Comment:
A lot of angular tutorials out there seem to just create and call an 'init()' method.